_docs-2-0/development/security.md

```diff
@@ -83,7 +83,8 @@ results sent back to the client.
 // user possesses both admin and system level access
 Authorizations auths = new Authorizations("admin","system");
 
-Scanner s = connector.createScanner("table", auths);
+AccumuloClient client = ... ;
```

Comment:
I don't think you need to show how to create client here, since you do it below.